### PR TITLE
Add Health Check Route

### PR DESCRIPTION
**Pull request by drake-yesware**
Adds a public health check route at /healthcheck to use for pings. Returns a 200 status and "OK".

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -81,6 +86,7 @@ func main() {
 		log.Printf("serving proxy at port %v\n", *port)
 	}
 
+	http.HandleFunc(newrelic.WrapHandleFunc(app, "/healthcheck", healthCheck))
```
Comment (mkostick) [inline main.go]: Is there any value to instrumenting this with New Relic?

Comment (drake-yesware) [reply main.go]: Not really, but I don't think it hurts to have it either.

Comment (drake-yesware) [reply main.go]: Actually, you're right. On second thought, thats going to skew the average response time metrics. Updated.

**Review by mkostick — APPROVED** [review]
G2G

**Pull request merged**